Maddori.Apple/Maddori.Apple/Screen/Setup/SetNickname/UIComponent/ProfileImageButton.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ProfileImageButton.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2023/01/21.
+//
+
+import UIKit
+
+import SnapKit
+
+final class ProfileImageButton: UIButton {
+    
+    // MARK: - property
+    
+    var profileImage = UIImageView(image: ImageLiterals.imgProfileNone)
```

Comment:
p2;

```suggestion
    private let profileImage = UIImageView(image: ImageLiterals.imgProfileNone)
```
수정해도 괜찮을 것 같네요 !

Reply:
아 이건 지금 제가 개발하고 있는 브랜치에서 확인할 수 있는건데요!
SetNicknameVC에서 프로필 이미지가 추가되면 profileImage의 image가 변경됩니다!
그래서 미리 var 로 선언해둔 겁니다!

Reply:
같은 크기의 component라면 
예를들어 `ImageLiterals.icTeamMananage`이 이미지로 변경한다 했을때, 
 `profileImage.image = ImageLiterals.icTeamMananage`이렇게 쓰면 ImageView 내의 image를 변경할 수 있습니다 ! 
이런 방법으로 변경하는 건 조금 다른 맥락일까요 ??

Reply:
profileImage의 image가 처음에는 `ImageLiterals.imgProfileNone`인데 
나중에는 `SetNicknameVC`에서 유저가 앨범에서 선택하거나 직접 촬영한 이미지로 변경되는 코드가 아래처럼 추가될 예정이라서

<img width="642" alt="스크린샷 2023-01-26 오후 11 02 03" src="https://user-images.githubusercontent.com/81340603/214855307-2e4835d5-af84-420a-a296-0e58e79982af.png">
<img width="604" alt="스크린샷 2023-01-26 오후 11 01 50" src="https://user-images.githubusercontent.com/81340603/214855322-90630086-9d74-463a-bf7a-a704825bc240.png">

미리 `private`을 빼고 `let`을 `var`로 바꿔둔 겁니다!

Reply:
네네 같은 맥락인 것 같아요 !
private를 떼는건 이해가 되는데
`profileImage.image = image as? UIIMage`가 된다면 `var`가 아닌 `let`으로 선언해도 괜찮을 것 같아요 !

Reply:
[1a649d6](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/279/commits/1a649d6249219eb5d4353cf50413c05ba213b08d)
아? 이미지가 바뀌어도 이미지 속성이 안 바뀐다면 let으로 선언해도 되는군요?
덕분에 오늘도 하나 배워갑니당 ㅎㅎ 감사해요 :)